### DISCUSSION TITLE
DC-ROMA%20I

### DISCUSSION THREAD

**Discussion by giscus** [fedora-riscv/fedorav-force-images-discussions#3] [Q&A]
# DC-ROMA%20I

http://127.0.0.1:3000/DC-ROMA%20I

**Comment by aahei** [comment]
Test ROMA1

**Comment by TechyLX** [comment]
1、系统初始化的显示比例过大

设置成150%的比例后，重启电脑，用户登录界面还是225%的显示比例

![7f23bce32479b2898fc8ec881bc8df6.jpg](https://prod-files-secure.s3.us-west-2.amazonaws.com/7bdf65e9-0afc-4b36-ac7c-3bbb5bc2636f/4960062c-275d-40d2-8155-7395fa19e1f5/7f23bce32479b2898fc8ec881bc8df6.jpg)

2、摄像头异常：打开相机APP，识别不到摄像头

![Screenshot from 2024-11-14 13-52-48.png](https://prod-files-secure.s3.us-west-2.amazonaws.com/7bdf65e9-0afc-4b36-ac7c-3bbb5bc2636f/ea8df149-b428-41f7-a041-622b99331e68/Screenshot_from_2024-11-14_13-52-48.png)

3、BT蓝牙功能异常：没有识别到蓝牙硬件

![image.png](https://prod-files-secure.s3.us-west-2.amazonaws.com/7bdf65e9-0afc-4b36-ac7c-3bbb5bc2636f/060fbb45-3a44-444b-8e51-d391ca46cb80/image.png)

4、播放器功能异常：播放视频提示异常

![Screenshot from 2024-11-14 14-08-52.png](https://prod-files-secure.s3.us-west-2.amazonaws.com/7bdf65e9-0afc-4b36-ac7c-3bbb5bc2636f/e3aa8bce-86f6-4406-b453-65040d739031/Screenshot_from_2024-11-14_14-08-52.png)


>

**Comment by EEddeenn** [comment]
尝试下载，提示 403 Forbidden 文件无下载权限
[文件链接](https://openkoji.iscas.ac.cn/pub/dist-repos/dl/DeepComputing/ROMA-I/images/latest/fedora-disk-gnome-workstation_deepcomputing_roma-sda.raw.gz)
